substrate/client/network/src/config.rs

```diff
@@ -620,6 +621,9 @@ pub struct NetworkConfiguration {
 	/// Configuration for the transport layer.
 	pub transport: TransportConfig,
 
+	/// Idle connection timeout.
```

Comment:
nit: Maybe we can add a bit more details here (maybe the DEFAULT_IDLE_CONNECTION_TIMEOUT docs)?

For litep2p, its important to note that we are considering the connection as idle if `idle_connection_timeout` or more passes between two substream open requests. When a substream is opened, the timeout is restarted.
If however the timeout is expired and the last subtream closes, the connection will be closed as well.

We had some plans of improvement in the past to consider the bytes sent over the substreams